docker-compose/docker-compose.yml

```diff
@@ -129,8 +129,8 @@ volumes:
   rabbitmq-volume:
   mssql-server-volume:
     external: true
-  #postgres-volume:
-    #external: true
+  postgres-volume:
+    external: true
```

Comment:
Perhaps this should stay commented out.

Reply:
Doesn't harm to have it uncommented. Commenting it out just reduces the effort to change the database provider